app/lib/frontend/templates/views/pkg/analysis/tool_env_info.mustache

```diff
@@ -0,0 +1,23 @@
+{{! Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+    for details. All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file. }}
+
+<div class="tool-env-info">
+  <h3>Analysis environment</h3>
```

Comment:
Should we perhaps use `<details><summary>` for this..

Like:

<details><summary>Analysis environment...</summary>

table....
</details>